GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -218,7 +218,6 @@ public IReadOnlyList<ObjectId> SelectedObjectIds
                 }
 
                 lock (_backgroundEvent)
-                lock (_graphModel)
```

Comment:
Is removing the locks the way to go?
Anything else to consider related to this?

Reply:
It should be safe:
- One thread is adding items.
- No thread is removing items.
- Multiple threads are reading.
- The lists/collections are now all of threadsafe types (sycnhronizedcollection & concurrentdictionary). This should make sure the count is reliable.
- Before the model is cleared, the other threads are killed.